docs/ocean/design_docs/prognostic-sediment-transport.rst

```diff
@@ -0,0 +1,519 @@
+	/>
+  </nml_record>
+
+
+Bedload transport also requires the definitions of the following variables:
+
+- Pre-defined sediment-related variables
+	
+  * sediment grain diameter: :code:`sedimentGrainDiameter(nSedimentClasses, nCells)`
+
+  * sediment grain density: :code:`sedimentGrainDensity(nSedimentClasses, nCells)`
+
+  * sediment class fraction on bed layer: :code:`sedimentClassFraction(nSedimentClasses)`
+
+  * bedload transport coefficient: :code:`sedimentBedloadCoefficient(nSedimentClasses)`
+
+- Other variables
+	
+  * bedload transport rate: :code:`sedimentBedloadTransportRate(nCells)`
+
+
+Implementation: Suspended sediment transport
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/19
+
+Contributors: Zhendong Cao
+
+The following namelist options should be defined:
+
+- A logical swith to turn ON/OFF suspended sediment transport: :code:`config_sediment_suspended`.
+
+- A character-type variable to define the empirical suspended transport method: :code:`config_sediment_suspended_formulation`. There are three options for it:
+	
+  * :code:`None`
+
+  * :code:`Zyserman-Fredsoe`
+
+  * :code:`Goldstein`
+
+  If this is set as :code:`None`, then the suspended transport is solved by the process-based approach.
+
+- A character-type variable to define the method to compute the sediment settling velocity: :code:`config_sediment_settling_formulation`. There are three options for it:
+
+  * :code:`None`
+
+  * :code:`Soulsby`
+
+  * :code:`Cheng`
+
+  If this is set as :code:`None`, then the sediment settling velocity is pre-defined manually.
+
+These can be defined in **Registry.xml** as follows:
+
+.. code::
+
+  <nml_record name="sediment_transport" mode="init;forward">
+        <nml_option name="config_sediment_suspended" type="logical" default_value=".false." units="unitless"
+                    description="Controls if sediment bedload transport is computed."
+                    possible_values=".true. or .false."
+        />
+        <nml_option name="config_sediment_suspended_formulation" type="chracter" default_value="None" units="unitless"
+                    description="Select the sediment bedload transport formulation"
+                    possible_values=" 'None', 'Zyserman-Fredsoe', 'Goldstein'"
+	/>
+  </nml_record>
+
+Suspended sediment transport also requires the definitions of the following variables:
+
+- Pre-defined sediment-related variables
+
+  * sediment settling velocity: :code:`sedimentSettlingVelocity(nSedimentClasses)`
+
+  * sediment critical erosion shear: :code:`sedimentCriticalErosionShear(nSedimentClasses)`
+
+  * sediment erodibility constant: :code:`sedimentErodibilityConstant(nSedimentClasses)`
+
+  * sediment bed porosity: :code:`sedimentBedPorosity(nSedimentClasses)`
+
+- Other variables
+
+  * sediment suspended concentration: :code:`sedimentSuspendedConcentration(nSedimentClasses, nCell)`
+
+Implementation: Morphological change
```

Comment:
This may not seem like it but it might be the hardest change to make.  You may want to meet us with me, @dengwirda, @cbegeman and @stephenprice to discuss this because it may relate to work we are doing for ice-shelf cavities.  Currently, `bottomDepth` is the (positive-down) bathymetry in MPAS-Ocean, but it isn't allowed to change in time.  It is also intimately tied to the z* vertical coordinate that is currently the default and is implicitly assumed in at least a few places in the code that @cbegeman has discussed with us (related to the computation of vertical velocities between layers).

A change to the model to allow `bottomDepth` to change with time could be a pretty big project and could potentially be a bit disruptive.  @matthewhoffman may also be interested in this work, since it also relates to his early career project.  In sort, this might be a big enough project to need a design document all on its own with input from a lot of different people.

Reply:
@caozd999, @lconlon and I have had a conversation about implementing morphological change to the bathymetry. Correct me if I'm wrong, but I think our conclusion was that this development wasn't strictly necessary for ICoM's goals and that it likely wouldn't be feasible in the [current development timeframe](https://icom.atlassian.net/wiki/spaces/ICOM/pages/101711907/Overview). I'd suggest removing it from this design doc and if we find ourselves ahead of the timeline or if this development seems necessary for ICoM then we create a new design document for morphological change alone.

Reply:
I'm trying to remember that conversation now....would erosion and deposition still work correctly if we don't change the bed elevation in the model? I remember that we decided that changing coastline morphology was out of the scope of this project, but I can't remember where we landed on bed elevation.

Reply:
@cbegeman you are right, we don't have to implement the morphological change component for ICoM now; I put it in this doc for a more general model component introduction.  As @xylar and @cbegeman suggested, I will remove morphological change from the design doc, and we can prepare a single design doc for when necessary. But it is good to know that this would influence/interest other people if we tend to do it; seems to me a necessary model development direction.

Reply:
@lconlon we would just assume the erosion and deposition are not strong enough to significantly change either the bed layer sediment mass or the sediment layer thickness. So it would work. And yes you are right - we had a discussion before and the coastline morphology won't be in the scope either.